tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -1425,12 +1437,20 @@ private function getSpecificPriceId(int $productId, int $combinationId, int $ord
 
     /**
      * @param string $productName
+     * @param Order|null $order
      *
-     * @return int
+     * @return FoundProduct
      */
-    private function getProductByName(string $productName)
+    private function getProductByName(string $productName, Order $order = null)
```

Comment:
```suggestion
    private function getProductByNameAndOrder(string $productName, Order $order = null)
```
Would it be possible to highlight in function name the fact it's filtered by order too?
Or maybe 2 functions (`getProductByName()` and `getProductInOrderByName()`) would be better?

Reply:
Fixed